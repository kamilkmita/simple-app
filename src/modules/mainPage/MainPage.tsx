import Container from "@material-ui/core/Container";
import messages from "../../constants/messages";
import ItemList from "./ItemList/ItemList";
import * as S from "./styles";

const MainPage = () => (
  <Container fixed>
    <S.Title>{messages.pageContentTitle}</S.Title>
    <ItemList />
  </Container>
);

export default MainPage;
