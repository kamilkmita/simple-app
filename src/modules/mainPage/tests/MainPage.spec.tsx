import renderer from "react-test-renderer";
import MainPage from "../MainPage";

describe("Mainpage", () => {
  test("proper render", () => {
    const component = renderer.create(<MainPage />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
