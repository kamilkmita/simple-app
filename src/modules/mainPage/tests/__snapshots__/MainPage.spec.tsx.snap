// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mainpage proper render 1`] = `
<div
  className="MuiContainer-root MuiContainer-fixed MuiContainer-maxWidthLg"
>
  <h1
    className="sc-bdvvaa"
  >
    Premier league clubs
  </h1>
  ItemListMockedComponent
</div>
`;
